src/storage/migration/util.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+from alembic import config, script
+from alembic.runtime import migration
+
+from storage.db_connection import AdminConnection
+
+alembic_cfg_path = Path(__file__).parent.parent.parent / 'alembic.ini'
+
+
+def db_needs_migration():
```

Comment:
I assume this module is where we need to fix the path-problem regarding starting fact from project root or from `src/` directory.

Reply:
should work now